machine/delta/delta_ag5648/installer.conf

```diff
@@ -0,0 +1,27 @@
+# Celestica Questone
+# CPU Module: Intel Broadwell
+
+description="DNI AG9032V1_CU with Intel Atom Rangeley CPU Module"
+
+# Default ONIE block device
+install_device_platform()
+{
+    for _device in /sys/block/*/device; do 
+
+        # work for kernel 4.1.17
+        if echo $(readlink -f $_device)|egrep -q "pci0000:00\/0000:00:1f.2"; then
+            _disk=`echo $_device | cut -f4 -d/`
```

Comment:
I am not a huge of using backticks -- please use the `$()` syntax for capturing the output of commands.